<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListaNiveles</class>
 <widget class="QDialog" name="ListaNiveles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Elija nivel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nivel1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nivel 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nivel2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nivel 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nivel3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nivel 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nivel4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nivel 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nivel5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nivel 5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nivel6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nivel 6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
